base/src/ColorConversionStrategy.h

```diff
@@ -0,0 +1,211 @@
+#include "ColorConversionXForm.h"
+
+class DetailAbstract
+{
+public:
+	DetailAbstract() {}
+	~DetailAbstract() {}
+	virtual void convert(frame_container& inputFrame, frame_sp& outFrame, framemetadata_sp outputMetadata) {};
```

Comment:
I dont understand why we need to remember iImg and oImg here ?
it is never a good idea to have unwanted state. It means one unwanted reference is always here.

Reply:
Addressed the change.